code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -914,3 +881,36 @@
 					to_chat(user, "<span class='warning'>You need a free hand for this rite!</span>")
 					uses += BLOOD_RECHARGE_COST // Refund the charges
 					qdel(rite)
+
+		if("Blood Spear (150)")
```

Comment:
These are ctrl+x -> ctrl+v with no changes to the content, right?

Reply:
I removed a couple of unneccesary backslashes, but other than that it's all the same.